src/bluesearch/k8s/create_indices.ipynb

```diff
@@ -0,0 +1,282 @@
+      "  _transport = transport_class(\n"
+     ]
+    }
+   ],
+   "source": [
+    "client = Elasticsearch(\n",
+    "    \"https://ml-elasticsearch.kcp.bbp.epfl.ch:443\",\n",
+    "    basic_auth=(\"elastic\", config('ES_PASS')),\n",
+    "    verify_certs=False,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "ObjectApiResponse({'name': 'elasticsearch-coordinating-0', 'cluster_name': 'elastic', 'cluster_uuid': 'dv5oiXnCRJiB8nhV7a81RA', 'version': {'number': '8.3.3', 'build_flavor': 'default', 'build_type': 'tar', 'build_hash': '801fed82df74dbe537f89b71b098ccaff88d2c56', 'build_date': '2022-07-23T19:30:09.227964828Z', 'build_snapshot': False, 'lucene_version': '9.2.0', 'minimum_wire_compatibility_version': '7.17.0', 'minimum_index_compatibility_version': '7.0.0'}, 'tagline': 'You Know, for Search'})"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create indices"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## mapping articles"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/tmp/ipykernel_1937749/4048237628.py:1: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.\n",
+      "  client.indices.create(\n",
+      "/home/reissant/miniforge3/envs/py10/lib/python3.10/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'ml-elasticsearch.kcp.bbp.epfl.ch'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings\n",
+      "  warnings.warn(\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "ObjectApiResponse({'acknowledged': True, 'shards_acknowledged': True, 'index': 'articles'})"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client.indices.create(\n",
+    "    index=\"articles\",\n",
+    "    body={\n",
+    "        \"settings\": {\"number_of_shards\": 2,\n",
+    "                     \"number_of_replicas\": 1},\n",
+    "        \"mappings\": {\n",
+    "            \"dynamic\": \"strict\",\n",
+    "            \"properties\": {\n",
+    "                \"article_id\": {\"type\": \"keyword\"},\n",
+    "                \"doi\": {\"type\": \"keyword\"},\n",
+    "                \"pmc_id\": {\"type\": \"keyword\"},\n",
+    "                \"pubmed_id\": {\"type\": \"keyword\"},\n",
```

Comment:
Just wondering:
- Do we plan to also specify the `arxiv_id`?
- Do we plan to keep the abstract into this table? It is going to be a duplicate of some part of the paragraphs table. But at the same time, maybe it can be useful at some point.

Reply:
1. we can add it as a field and only use it when available, it's easy and does not create any issue
2. I think it is useful to have the abstract on the articles table because if you get a match on any part of the paper, you can always easily retrieve the abstract from the article ID. What do you think?

Reply:
Sounds good! Thank you for your reply!